sdk/src/types/block/output/basic.rs

```diff
@@ -320,6 +320,14 @@ impl BasicOutput {
 
         None
     }
+
+    /// Checks whether the basic output is an implicit account.
+    pub fn is_implicit_account(&self) -> bool {
+        self.unlock_conditions.len() == 1
```

Comment:
any unlock condition does suffice?

Reply:
You can't build a BasicOutput without an address so if there is only one UC, it has to be an address

Reply:
sorry, wasn't clear enough, the question was whether we should be explicit here rather then just checking that there is a UC (which then has to be an address UC)